src/OrchardCore.Modules/OrchardCore.Workflows/Trimming/Startup.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.BackgroundTasks;
+using OrchardCore.DisplayManagement.Handlers;
+using OrchardCore.Environment.Shell.Configuration;
+using OrchardCore.Modules;
+using OrchardCore.Navigation;
+using OrchardCore.Security.Permissions;
+using OrchardCore.Settings;
+using OrchardCore.Workflows.Trimming.Drivers;
+using OrchardCore.Workflows.Trimming.Services;
+
+namespace OrchardCore.Workflows.Trimming;
+
+[Feature("OrchardCore.Workflows")]
```

Comment:
Is it intended to be a new feature? If not, maybe move these services into the WF startup file instead of having multiple startups for the same service. Or, just create Workflow.Trimming feature

Reply:
From the docs, you mention feature. So probably create OrchardCore.Workflows.Trimming feature

Reply:
In the docs I used that as "functionality". I don't think this should be its own feature, since it can already be disabled and (just as for Audit Trail), I'd highly recommend it keeping it on everywhere.

I moved these to the root `Startup`.